docs/platform-support.md

```diff
@@ -3,3 +3,4 @@
 FakeItEasy officially supports the following platforms:
 
 * .NET Framework 4.0 onwards
+* .NET Standard 1.6

```

Comment:
I don't mind it as it is. The .NET Standards are built so that "onwards" is implied. So far the Framework 4.xes are backward compatible, but do we know that it'll continue?

But really, I wouldn't fight against either change you proposed, if you're passionate about it.